resources/dicts/events/death/general.json

```diff
@@ -5634,385 +5923,414 @@
+        },
+        "history:": [
+            {
+                "cats": ["m_c"],
+                "reg_death": "m_c died when r_c lured {PRONOUN/m_c/object} into a Twoleg trap.",
+                "lead_death": "{VERB/m_c/were/was} lured by r_c into a Twoleg trap"
+            }
+        ]
+    },
+    {
+        "event_id": "gen_death_angrymurder1",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["murder"],
+        "tags": ["all_lives", "low_lives"],
+        "weight": 30,
+        "event_text": "r_c hisses {PRONOUN/r_c/poss} grievances into m_c's ear as {PRONOUN/r_c/subject} {VERB/r_c/kill/kills} m_c.",
+        "m_c": {
+            "age": ["adolescent", "young adult", "adult", "senior adult", "senior"],
+            "dies": true
+        },
+        "r_c": {
+            "age": ["adolescent", "young adult", "adult", "senior adult", "senior"],
+            "trait": ["vengeful", "arrogant", "rebellious", "troublesome", "fierce"]
+        },
+        "history:": [
+            {
+                "cats": ["m_c"],
+                "reg_death": "m_c died when r_c lured {PRONOUN/m_c/object} into a Twoleg trap.",
+                "lead_death": "{VERB/m_c/were/was} lured by r_c into a Twoleg trap"
+            }
+        ]
+    },
+    {
+        "event_id": "gen_death_starclanmurder1",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["murder"],
+        "tags": ["lives_remain"],
+        "weight": 30,
+        "event_text": "r_c attempts to take what remains of m_c's lives, but a warning from StarClan gives {PRONOUN/m_c/object} enough information to avoid such a fate. {PRONOUN/m_c/subject} still lose a life, but at least {PRONOUN/m_c/subject} manage take r_c down with {PRONOUN/m_c/object}.",
+        "m_c": {
+            "status": ["leader"],
+            "skill": ["OMEN,1", "DREAM,1", "PROPHET,1", "GHOST,2", "STAR,1"],
+            "dies": true
+        },
+        "r_c": {
+            "age": ["young adult", "adult", "senior adult", "senior"],
+            "dies": true
+        },
+        "history:": [
+            {
+                "cats": ["m_c"],
+                "lead_death": "{VERB/m_c/were/was} lured by r_c into a Twoleg trap"
+            },
+            {
+                "cats": ["r_c"],
+                "reg_death": "r_c died when {PRONOUN/r_c/subject} tried to kill m_c and failed."
+            }
+        ]
+    },
+    {
+        "event_id": "gen_death_war_predatorsaved",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "tags": ["no_body", "all_lives"],
+        "weight": 20,
+        "event_text": "m_c acts without thinking, saving the life of a o_c_n warrior by distracting a predator - which instead turns on m_c.",
+        "m_c": {
+            "status": ["warrior", "deputy", "leader", "medicine cat", "medicine cat apprentice", "apprentice"],
+            "age": ["adolescent", "young adult", "adult", "senior adult"],
```

Comment:
in this case you could omit the age parameter.  the status parameter already constrains the cats to those statuses, which consequently will also constrain it to the ages those statuses are allowed to be.  notably, with the current set up, senior medicine cats and leaders would be incorrectly excluded.  you could just add senior to the age parameter to fix this, but i find it better to simply omit the age parameter to entirely avoid forgetting some edge case age.

please carry this suggestion on to the other events you've added in this PR